extensions-contrib/prometheus-emitter/src/main/resources/defaultMetrics.json

```diff
@@ -1,5 +1,5 @@
 {
-  "query/time" : { "dimensions" : ["dataSource", "type"], "type" : "timer", "conversionFactor": 1000.0, "help":  "Seconds taken to complete a query."},
+  "query/time" : { "dimensions" : ["dataSource", "type"], "type" : "timer", "conversionFactor": 1000.0, "histogramBuckets": [0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0, 7.5, 10.0, 30.0, 60.0, 120.0, 300.0], "help":  "Seconds taken to complete a query."},
```

Comment:
Why override this? Isn't this already the default value?

Reply:
Yup, it is the default value. My rationale behind overriding this value was that users will find an example to work with. 
@capistrant, do you have any example metric in mind that we can use to show a sample for override?

Reply:
@ashwintumma23 `task/run/time` is an example of a metric that we found is not very useful with the defaults

Reply:
Thanks @capistrant ! What do you think the default values can be? I am thinking `[1.0, 2.5, 5.0, 7.5, 10.0, 30.0, 60.0, 120.0, 300.0, +Inf]` to capture . But again, this is only a sample use case.

On another thought, do we need to specify an example default value here, or let users/ cluster administrators configure it as they see fit?

Reply:
I think only administrators will truly know the proper values to put. If we show an example of how to override, I think that this metric is a good choice though as it should make sense to operators. I would personally add `600, 1200, 1800` on to the end of the override, but that is just based off of me and how we have longer running batch jobs that I'd prefer don't all get tossed into inf.

Reply:
Makes sense! A batch job's time will be totally dependent on the use case. As we have already documented how to configure the parameter, is it fine if we don't show it via an example in the `defaultMetrics.json` file?

Reply:
> ya, I'm ok with us leaving it to the operator to decide if/when they want to take advantage of this change

Sounds good, removing the example from this pull request in the next commit.